enterprise/server/vmexec/vmexec.go

```diff
@@ -322,11 +361,45 @@ func newCommand(start *vmxpb.ExecRequest) (*command, error) {
 	}, nil
 }
 
+func (c *command) logDebugInfo(err error) {
```

Comment:
Should we do some error filtering here? There can be lots of uninteresting reasons a command fails

Reply:
Done, filtering to "exec format error" for now